builtin/mainmenu/dlg_delete_world.lua

```diff
@@ -17,39 +17,37 @@
 
 
 local function delete_world_formspec(dialogdata)
-
 	local retval =
-		"size[11.5,4.5,true]" ..
-		"label[2,2;" ..
+		"size[10,2.5,true]" ..
+		"label[0.5,0.5;" ..
 		fgettext("Delete World \"$1\"?", dialogdata.delete_name) .. "]" ..
-		"button[3.25,3.5;2.5,0.5;world_delete_confirm;" .. fgettext("Delete") .. "]" ..
-		"button[5.75,3.5;2.5,0.5;world_delete_cancel;" .. fgettext("Cancel") .. "]"
+		"button[0.5,1.5;2.5,0.5;world_delete_confirm;" ..
+		minetest.colorize("#FFFF00", fgettext("Delete")) .. "]" ..
+		"button[7.0,1.5;2.5,0.5;world_delete_cancel;" .. fgettext("Cancel") .. "]"
 	return retval
 end
 
 local function delete_world_buttonhandler(this, fields)
 	if fields["world_delete_confirm"] then
-
 		if this.data.delete_index > 0 and
 			this.data.delete_index <= #menudata.worldlist:get_raw_list() then
```

Comment:
One more tab indent

Reply:
@paramat Not necessary. The line above or below wasn't touched.

Reply:
Optional yes, but seeing as code cleanup is being done in this area i can't see a reason to not do this.

Reply:
Done.